contracts/mock/MockRewardsDistributor.sol

```diff
@@ -6,9 +6,9 @@ import "../feirari/IRewardsDistributorAdmin.sol";
 import "@openzeppelin/contracts/access/Ownable.sol";
 
 contract MockRewardsDistributor is IRewardsDistributorAdmin, Ownable {
-    event SuccessSetAdmin(bool set);
-    event SuccessAcceptPendingAdmin(bool set);
-    event SuccessGrantComp(bool set);
+    event SuccessSetAdmin(bool set, address pendingAdmin);
```

Comment:
the bool set provides no information, is it ever false?

Reply:
No. I will remove it.

Reply:
done